Client.xcodeproj/project.pbxproj

```diff
@@ -5826,7 +5826,7 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
 				APPLICATION_EXTENSION_API_ONLY = YES;
-				CODE_SIGN_ENTITLEMENTS = "$(SRCROOT)/Extensions/Entitlements/Fennec.entitlements";
+				CODE_SIGN_ENTITLEMENTS = "$(inherited)Extensions/Entitlements/Fennec.entitlements";
```

Comment:
@st3fan, with this `$(inherited)` change I passed the error we were having, now there are issues with the Codesigns `[Failed to find Codesign Groups](https://app.bitrise.io/build/f179616a38e26400)` 
Investigating...